applications/GeoMechanicsApplication/custom_strategies/schemes/README.md

```diff
@@ -29,33 +54,55 @@ $$\dot{x}\_{t + \Delta t} = (x\_{t + \Delta t} - x\_{t} ) / \Delta t$$
-
-## Dynamic and damped schemes
-The dynamic and damped schemes are both Generalized Newmark schemes (as seen in the class diagrams) and therefore update the derivatives using the same equations, using the same `UpdateVariablesDerivatives` function.
-
-However, `CalculateSystemContributions`, `CalculateRHSContribution` and `CalculateLHSContribution` are overridden and add extra terms to the system matrix and right hand side vector. The damped scheme only adds damping terms, while the dynamic scheme adds both damping and mass terms, to also take into account the inertia of the system.
-
-One more anomaly is the `Predict` function, currently found in the `NewmarkDynamicUPwScheme` class. This functionality should move to the `GeneralizedNewmarkScheme` class, since it is not specific to the dynamic scheme.
+$$\dot{x}\_{t + \Delta t} = \dot{x}\_{t} + (1 - \gamma)\Delta t \ddot{x}\_{t} + \gamma \Delta t \ddot{x}\_{t + \Delta
+t}$$
+
+Note that the order here is the other reversed with respect to the Backward Euler scheme. This is because
+$\ddot{x}\_{t + \Delta t}$ is used in the equation for $\dot{x}\_{t + \Delta t}$.
+
+For these functions, identically to the Backward Euler scheme, $\ddot{x}$ and $\dot{x}$ refer to the first and second
+derivatives of $x$, while subscripts $t + \Delta t$ and $t$ refer to the end and start of the current time step. The
+magnitude of the time step is denoted by $\Delta t$. The parameters $\theta$, $\gamma$ and $\beta$ are the parameters of
+the Generalized Newmark scheme, with the following conditions: $0\le\theta\le 1$, and $0\le\gamma\le 1$ and
+$0\le\beta\le 0.5$.
+
+For setting the time factors (done in the `SetTimeFactors` override), the following equations are used. For the scalar
+variables, the time coefficient is set to $1/(\theta\Delta t)$, while for the vector variables, the time coefficients
+are set to $\gamma / (\beta \Delta t)$.
+
+## Dynamic, damped and quasistatic Generalized Newmark schemes
+
+The dynamic, damped and quasistatc schemes are all Generalized Newmark schemes (as seen in the class diagrams). For the
+vector variables part (displacements and/or rotations) they take different parts of the equation of motion into account.
+Quasistatic the contribution to matrices and vectors for displacements/rotations only, damped the contribution to
+matrices and vectors for displacements/rotations and velocities/angular velocities and dynamic the contribution to
+matrices and vectors for accelerations/angular accelerations, velocities/angular velocities and displacements/rotations.
+Therefore the functionality `UpdateVariablesDerivatives`, `CalculateSystemContributions`, `CalculateRHSContribution`
+and `CalculateLHSContribution` are overridden and add extra terms to the system matrix and right hand side vector.
```

Comment:
Since this section now also includes quasistatic, I think it's good to be explicit that adding the extra terms is only done for dynamic and damped:
```suggestion
and `CalculateLHSContribution` are overridden and add extra terms to the system matrix and right hand side vector for the dynamic/damped schemes.
```

Reply:
done